### DISCUSSION TITLE
Help me please, Cefsharp Settings.

### DISCUSSION THREAD

**Discussion by KairoMartins18** [cefsharp/CefSharp#3349] [Q&A]
the error happens in the line Cef.Initialize (settings);

error!! CEF can only be initialized once per process. This is a limitation of the underlying CEF/Chromium framework. You can change many (not all) settings at runtime through RequestContext.SetPreference. See https://github.com/cefsharp/CefSharp/wiki/General-Usage#request-context-browser-isolation

if you need any information on what I'm trying to do:[here](https://github.com/cefsharp/CefSharp/wiki/General-Usage#useragent)

**Comment by amaitland** [comment]
For background please read https://github.com/cefsharp/CefSharp/wiki/General-Usage#initialize-and-shutdown

As the exception says you can only call Cef.Initialize once, if you are attempting to change the user agent multiple times then you need to use a different method. 

Make sure Cef.Initialize is called before you create your first ChromiumWebBrowser instance. 

If you require further assistance then please provide a clear description of what you are trying to achieve and a detailed example of your code.
